app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter(post_id)
+    post = Post.find_by(id: post_id)
+    post.likes_counter = post.likes.count
+    post.save
+  end
```

Comment:
- [OPTIONAL] This can be refactored as follows to economize lines of code.
```suggestion
  def update_likes_counter()
    post.increment!(:likes_counter)
  end
```

Rails will take care of finding the post for you!